jsk_pcl_ros/launch/openni2_remote.launch

```diff
@@ -14,6 +14,7 @@
   <arg name="uv_scale" default="1.0"  />
   <arg name="depth_calibration_file" default="$(find jsk_pcl_ros)/config/default_depth_calibration_parameter.yaml" />
   <arg name="republish" default="false" />
+  <arg name="use_warn" default="true" />
```

Comment:
I think that setting default true changes the log output from current. Why do you want to set default use_warn true? Is there some important warning message in ImageResizer?

Reply:
OK. I modified not change default value (use_warn is false).